_posts/technotes/2017-08-18-tesseract-16.md

```diff
@@ -0,0 +1,118 @@
+---
+name: tesseract
+layout: post
+title: "Tesseract and Magick: High Quality OCR in R"
+date: 2017-08-18
+authors:
+  - name: Jeroen Ooms
+    url: https://ropensci.org/about/
```

Comment:
don't need to add a URL for yourself unless you want it to be something different from ropensci.org/about#staff